### PR TITLE
Fix: App architecture and code quality

### PR DESCRIPTION
**Pull request by moiskillnadne**

### CODE REVIEW

File: src/core/interfaces/index.ts [hunk 1]
```diff
@@ -163,19 +163,19 @@ export interface IActivityFlow {
...
-export interface IUser {
+export interface User {
   secretId: string
   nickname: string
   firstName: string | null
   lastName: string | null
 }
 
-export interface IResponse {
+export type ActivityItemReponse = {
```
Comment (karser) [inline src/core/interfaces/index.ts:173]: reSponse

File: src/core/interfaces/index.ts [hunk 2]
```diff
@@ -163,19 +163,19 @@ export interface IActivityFlow {
...
   activityId: string
-  data: IResponseItem[]
+  data: ItemReponse[]
 }
 
-export interface IResponseItem {
+export type ItemReponse = {
```
Comment (karser) [inline src/core/interfaces/index.ts:178]: reSponse

**Review by karser — CHANGES_REQUESTED** [review]

File: src/core/interfaces/index.ts (hunk 1 above)
Comment (moiskillnadne) [reply src/core/interfaces/index.ts]: Done

File: src/core/interfaces/index.ts (hunk 2 above)
Comment (moiskillnadne) [reply src/core/interfaces/index.ts]: Done

**Pull request merged**